packages/worker/src/runtime.ts

```diff
@@ -56,17 +55,29 @@ export interface RuntimeOptions {
 
 export interface CompiledRuntimeOptions {
   shutdownSignals: NodeJS.Signals[];
-  telemetryOptions: TelemetryOptions;
+  telemetryOptions: CompiledTelemetryOptions;
   logger: Logger;
 }
 
-function defaultTelemetryOptions(): TelemetryOptions {
-  return {
-    tracingFilter: 'temporal_sdk_core=INFO',
-    logging: {
-      console: {},
-    },
-  };
+/**
+ * A helper to build a filter string for use in `RuntimeOptions.telemetryOptions.tracingFilter`.
+ *
+ * Example:
+ *  ```
+ *  telemetryOptions: {
+ *    logging: {
+ *     filter: makeTracingFilterString('DEBUG', 'INFO');
+ *     // ...
+ *    },
+ *    tracing: {
+ *     filter: makeTracingFilterString('DEBUG', 'INFO');
+ *     // ...
+ *    }
+ *  }
+ * ```
+ */
+export function makeTracingFilterString(coreLevel: native.LogLevel, otherLevel: native.LogLevel = 'WARN'): string {
```

Comment:
nit: use a single options arg instead of 2 separate args so it's easier to understand what each level refers to.